integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/createTable/TestCreateDDLForComplexMapType.scala

```diff
@@ -482,8 +482,8 @@ class TestCreateDDLForComplexMapType extends QueryTest with BeforeAndAfterAll {
        """.stripMargin)
     sql("INSERT INTO carbon SELECT * FROM carbon")
     checkAnswer(sql("select * from carbon"), Seq(
-      Row(Map(1 -> "Nalla", 2 -> "Singh", 4 -> "Kumar")),
-      Row(Map(1 -> "Nalla", 2 -> "Singh", 4 -> "Kumar")),
+      Row(Map(1 -> "Gupta", 2 -> "Singh", 4 -> "Kumar")),
+      Row(Map(1 -> "Gupta", 2 -> "Singh", 4 -> "Kumar")),
```

Comment:
I don't get the changes of this file? why changing of name ? :)

Reply:
Actually the earlier testcases were based on Map having the older values, now the implementation is being changed to override the older values in the map with the new values that come after.

Reply:
@ravipesala please check